helm/charts/oathkeeper/Chart.yaml

```diff
@@ -25,5 +25,5 @@ dependencies:
   - name: oathkeeper-maester
     version: 0.4.11
     condition: maester.enabled
-    alias: maester
```

Comment:
I think this could break something in the internal mapping, not sure. Have you tested this change in Kubernetes?

Reply:
I've installed the hydra & oathkeeper charts both with maester with this change and they run with no problems.
Also the logs do not show any obvious errors, so this seems to work :) 
Though I cannot be sure it is not breaking anything internally, as I did not do extensive testing beyond the mentioned steps

Reply:
Ok, should be good then! :)